Makefile

```diff
@@ -1,28 +1,25 @@
-all: vendor
+all:
 	go build -o cni-noop ./cmd/cni-noop
 	go build -o cnispawn ./cmd/cnispawn
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
-glide:
-	@which glide || go get -u github.com/Masterminds/glide
-
+vendor: glide.lock | glide
+	glide --quiet install --strip-vendor
 glide.lock: glide.yaml | glide
 	glide update --strip-vendor
+glide:
+	@which glide || go get -u github.com/Masterminds/glide
 
-vendor: glide.lock | glide
-	glide install --strip-vendor
-
-.PHONY: glide vendor all
-
-clean:
-	rm -rf ./{kubeadm-nspawn,cni-noop,cnispawn}
+.PHONY: vendor all clean
 
+clean: clean-bins clean-rootfs clean-cni clean-image clean-ssh-keys
+clean-ssh-keys:
+	rm -rf ./id_rsa*
+clean-bins:
+	rm -rf ./{cni-noop,cnispawn,kubeadm-nspawn}
 clean-rootfs:
 	sudo rm -rf kubeadm-nspawn-*
-
 clean-cni:
-	sudo rm -rf /var/lib/cni/networks/mynet/*
-	sudo iptables-restore < ~/iptables
-
+	sudo rm -rf /var/lib/cni/networks/mynet
```

Comment:
I put that there because every time we are running `cnispawn` it creates a file this folder and increments the ip addresses of the spawned containers.

How are we going to clean this up instead?